config.js

```diff
@@ -427,25 +427,12 @@ var config = {
 
     // Specify the settings for video quality optimizations on the client.
     // videoQuality: {
-    //    // Provides a way to prevent a video codec from being negotiated on the JVB connection. The codec specified
```

Comment:
Please put the dreprecated ones at the end of video quality with a DEPRECATED title, see how we've done that with other options.